### PR TITLE
add kerasStyleAPIGuide

### PR DESCRIPTION
**Pull request by simanw304**
add SparseDense & SparseEmbedding doc

### CODE REVIEW

File: docs/docs/KerasStyleAPIGuide/Layers/embedding.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+## **Embedding**
```
Comment (zhichao-li) [inline docs/docs/KerasStyleAPIGuide/Layers/embedding.md]: SparseEmbedding

**Comment by zhichao-li** [comment]
Jenkins？

**Comment by simanw304** [comment]
http://10.239.47.211:18888/job/ZOO-PR-Validation/878/
jenkins passed

**Comment by zhichao-li** [comment]
cc @Deliadd

**Pull request merged**